RECOVERY.md

```diff
@@ -19,24 +19,29 @@ For example, when using [electrum](https://electrum.org/):
 
 When swapping funds to a `lightning-kmp` wallet, the following steps are performed:
 
-- funds are sent to a swap-in address via a swap transaction
+- funds are sent to a swap-in address via a swap transaction. 
 - we wait for that transaction to have enough confirmations
 - then, if the fees don't exceed the user's liquidity policy, these funds are moved into a lightning channel
 
+We use musig2 to aggregate user keys (user being the wallet) and server keys (server being the LSP: the ACINQ node): swap-in addresses are standard p2tr addresses, and
+swap-in transactions to your wallet are indistinguishable from other p2tr transactions.
+
 The swap transaction's output can be spent using either:
 
-1. A signature from the user's wallet and a signature from the remote node
+1. An aggregated musig2 signature built from a partial signature from the user's wallet and a partial signature from the remote node
 2. A signature from the user's wallet after a refund delay
 
 Funds can be recovered using the second option and [Bitcoin Core](https://github.com/bitcoin/bitcoin).
-This process needs at least Bitcoin Core 25.0.
+This process needs at least Bitcoin Core 26.0.
 
 This process will become simpler once popular on-chain wallets (such as [electrum](https://electrum.org/)) add supports for output script descriptors.
 
-### Extract master keys
+### Get your wallet descriptor
```

Comment:
We need to keep documenting the legacy p2wsh case for the foreseeable future. I'm unclear on the differences between the p2wsh/p2tr cases and what can be factored. Also the address rotation adds another layer of complexity and makes factoring the doc even more necessary. Is the difference just around descriptors?

Reply:
I added a link to the recovery documentation for legacy swap-ins. 

The new recovery process is simpler, and just requires to import a single descriptor into bitcoin core (26 or later). This process will remain the same once address rotation has been added (the current descriptor will match all generated addresses).

The biggest difference is that the legacy descriptor's path included an xpub with a path that started at the master key, which required an intermediary step to get the xpriv first with Electrum. Here the descriptor includes an xpriv for the swap-in refund keys (it cannot be used to go "up").